src/tokens/size/spacing.js

```diff
@@ -46,5 +46,13 @@ module.exports = {
         gap: { value: '{size.spacing.gap.inline.small.value}' },
       },
     },
+    button_group: {
+      gap: {
```

Comment:
This looks like the wrong hierarchy: Tokens should start broad and get more specific. size → spacing → gap → button_group

Reply:
Done! https://github.com/cloudfour/cloudfour.com-patterns/pull/1240/commits/d98fb22ad25653c9a5223a2b5f27e8f2f0c2bfef